daemon/container_unix.go

```diff
@@ -914,6 +914,12 @@ func (container *Container) allocateNetwork() error {
 		if mode.IsDefault() {
 			networkName = controller.Config().Daemon.DefaultNetwork
 		}
+		if mode.IsUserDefined() {
+			if n, err := container.daemon.FindNetwork(networkName); err == nil {
+				networkName = n.Name()
```

Comment:
@aboch I understand. but it is strange in this case to not handle the error.
I personally would prefer to handle the error and return right here.